packages/global/scss/index.scss

```diff
@@ -709,6 +713,11 @@ body {
 .section-card-block {
   &__header {
     @include theme-card-header();
+    font-size: 32px;
+    font-weight: 800;
+    line-height: .5;
```

Comment:
I would ask Ashton the actual line height value to use instead of "decrease by 1pt"